labextension/src/widgets/deploys-progress/KatibProgress.tsx

```diff
@@ -86,32 +122,89 @@ export const KatibProgress: React.FunctionComponent<IKatibProgressProps> = props
   };
 
   const getComponent = (experiment: IKatibExperiment) => {
-    let IconComponent: any = UnknownIcon;
-    let iconColor = '#5f6368';
+    let tooltipSet = false;
+    let styles = { color: '#5f6368', height: 18, width: 18 };
+    let IconComponent = <UnknownIcon style={styles} />;
 
     switch (experiment.status) {
```

Comment:
It looks to me like the two cases `FAILED` and `SUCCEEDED` implement the same logic. Maybe we could move it a separate function?